src/force/force.ts

```diff
@@ -444,7 +444,7 @@ export class Force extends BaseContract {
           account_id: this.effectAccount.vAccountRows[0].id,
           batch_id: batchPk,
           num_tasks: content.tasks.length,
-          sig: null
+          sig: isBscAddress(fromAccount) ? vaccSig.toString() : null,
```

Comment:
why do we reuse the vaccSig from vtransfer for publishbatch? Does it even need a sig?

Reply:
Yiu are right we don't need it, I saw it on the whitelist in the relayer with sig required, but that was a mistake